dist/tools/openocd/openocd.sh

```diff
@@ -308,7 +308,7 @@ do_debugserver() {
             -c 'gdb_port ${GDB_PORT}' \
             -c 'init' \
             -c 'targets' \
-            -c 'halt'"
+            -c 'reset halt'"
```

Comment:
Instead of this, maybe we could reuse `${OPENOCD_DBG_START_CMD}` like in `do_debug`. This allows boards requiring this to override start command.